assets/pages/contributions-landing-eu/components/contributionSelectionContainer.js

```diff
@@ -0,0 +1,33 @@
+// @flow
```

Comment:
As far as I can tell, this is a copy/paste from contributionSelectionContainer from UK or US. Is it possible to reuse that component in some way? Maybe we can create a `contributionsComponents` folder inside `components` and put all these contributions-shared components there.

Reply:
Agreed. There is scope to share some things and I've been discussing that with Jamie. We're in a state of flux as we transition from one set of pages and designs to another, so I'm hoping we can find some common ground when we've settled these things down.